### DISCUSSION TITLE
Embedding NSFWJS into a browser like SPIN

### DISCUSSION THREAD

**Discussion by hi337** [infinitered/nsfwjs#650] [Ideas]
Hey,

   I was wondering about the possibility of embedding this wonderful software into a clean browser like the SPIN Browser. 

  Its a great browser built on top of Firefox and it only has one problem. It only puts search engines in safe mode, which still can show racy images. And adding something similar to [nsfw-filter](https://github.com/nsfw-filter/nsfw-filter)'s functionality would do the app wonders.

  I can't say that I know Firefox development, nor am I that good at node, but implementing something like this into a browser, making it non removable (unlike an extension) would be really cool. 

Thanks for your time

**Comment by GantMan** [comment]
I def think it's a cool idea.  It's totally doable, I just can't do it myself.
